README.rst

```diff
@@ -186,11 +186,11 @@ impact to other applications will be minimal.
 Upgrading an existing Synapse
 =============================
 
-The instructions for upgrading synapse are in `UPGRADE.rst`_.
+The instructions for upgrading synapse are in `the upgrade notes`_.
 Please check these instructions as upgrading may require extra steps for some
 versions of synapse.
 
-.. _UPGRADE.rst: UPGRADE.rst
+.. _the upgrade notes: https://matrix-org.github.io/synapse/develop/upgrading
```

Comment:
```suggestion
.. _the upgrade notes: https://matrix-org.github.io/synapse/develop/docs/upgrade.md
```

I think?

Reply:
mdbook operates from the perspective of the `docs` folder, so I believe if we put the file at `docs/upgrade.md` then the resulting URL will be `https://matrix-org.github.io/synapse/develop/upgrade.html`.

We can lob off the `.html` bit though.